crates/kubelet/src/config.rs

```diff
@@ -71,12 +71,16 @@ impl Config {
             default_node_ip(&mut hostname.clone(), &addr)
                 .expect("unable to get default node IP address")
         });
+        let node_name = match opts.node_name {
```

Comment:
Is it clearer to write? 
```rust
let node_name = opts.node_name.unwrap_or_else(|| sanitize_hostname(&hostname));
```
FWIW I'm fine with the match expression

Reply:
agreed